### ISSUE TITLE
detect  nothing

### ISSUE THREAD

**Issue by henbucuoshanghai** [mystic123/tensorflow-yolo-v3#75]
W tensorflow/core/framework/allocator.cc:124] Allocation of 18874368 exceeds 10% of system memory.
/home/ubuntu/tensorflow-yolo-v3/utils.py:215: RuntimeWarning: invalid value encountered in less
  iou_mask = ious < iou_threshold
Predictions found in 0.48s

and the output img is the same sa input ,it detect nothing?
why???   i have no gpu

**Comment by henbucuoshanghai** [comment]
tiny net

**Comment by henbucuoshanghai** [comment]
the detected_boxes are as follows:   
[[[nan nan nan ... nan nan nan
  ...
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]]]

i dont know why.

**Comment by henbucuoshanghai** [comment]
OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key detector/yolo-v3-tiny/Conv/BatchNorm/beta not found in checkpoint

**Comment by mostafiz76** [comment]
yes for me also it detects nothing, it shows prediction in 5ms but the output image shows nothing

**Comment by henbucuoshanghai** [comment]
nothing。。

**Comment by mostafiz76** [comment]
nothing in the sense no bounding boxes, the output image is same as the input image

**Comment by 1984chaowang** [comment]
same for me after followed the steps
iou_mask = ious < iou_threshold
